pkg/patcher/extender_readiness.go

```diff
@@ -146,8 +146,13 @@ func (p *SchedulerPatcher) UpdateReadinessConfigMap(ctx context.Context, csi *cs
 		return err
 	}
 
-	readinessStatuses, err := p.updateReadinessStatuses(ctx, options.kubeSchedulerLabel, cmCreationTime,
-		useOpenshiftSecondaryScheduler)
+	var readinessStatuses *ReadinessStatusList
+	if useOpenshiftSecondaryScheduler {
+		readinessStatuses, err = p.updateReadinessStatusesForOpenshiftSecondaryScheduler(ctx,
+			options.kubeSchedulerLabel, cmCreationTime, 10*time.Second, 12)
+	} else {
+		readinessStatuses, err = p.updateReadinessStatuses(ctx, options.kubeSchedulerLabel, cmCreationTime)
```

Comment:
looks like no change for non-rke2 platform for this logic ?

Reply:
> looks like no change for non-rke2 platform for this logic ?

Yes, actually this logic for Openshift platform without secondary scheduler, i.e. Openshift 4.9 and lower, eventually remains unchanged. And, this func will be re-used in the check for Openshift platform with secondary scheduler, i.e. check every 10s until the secondary scheduler ready with scheduler extender config applied or 2-min timeout hit.